### DISCUSSION TITLE
Is it possible to get the name of the parent folder of Obsidian Vault

### DISCUSSION THREAD

**Discussion by hidara2000** [SilentVoid13/Templater#774] [Help] [answered]
Thanks for the great plugin

I have multiple obsidian vaults. Is there any way to get the vault(parent folder) name?

I would like to automatically fill vault_name=Android as shown below

![image](https://user-images.githubusercontent.com/15170494/184064766-35b478c8-8cde-4b3c-93f1-2d73c8081e87.png)

Thank you

**Comment by AB1908** [comment]
Not a stable API IIRC but look into `app.vault.adapter.basePath`.

**Comment by AB1908** [comment, marked as answer]
Disregard, use [`app.vault.getName()`](https://github.com/obsidianmd/obsidian-api/blob/3770989c2364ead5429eb7e4475e095588bbe914/obsidian.d.ts#L1161).

**Reply by hidara2000** [reply]
Perfect. Thank you!
